### ISSUE TITLE
Version 1.3.0 breaks my install 

### ISSUE THREAD

**Issue by pelmered** [joserick/laravel-livewire-discover#8]
When I installed version `1.3.0` my whole install broke with this error: 

```
Laravel\Pulse\PulseServiceProvider::Laravel\Pulse\{closure}(): Argument #1 ($livewire) must be of type Livewire\LivewireManager, Joserick\LaravelLivewireDiscover\LaravelLivewireDiscoverManager given, called in /vendor/laravel/framework/src/Illuminate/Support/ServiceProvider.php on line 309
```

Everything works fine if I downgrade to `1.2.0` or `1.2.1`.

Now, when I look at the code. I see that `LaravelLivewireDiscoverManager` no longer extends `LivewireManager` as it did in `1.2.1`. This is obviously a breaking change.

**Comment by joserick** [comment]
Hi,

Thank you very much for reporting the error, it has been fixed in version 1.3.1.

Now, the issue with this change, was based on the implementation of the idea of ​​a decorator as mentioned in the Laravel documentation [extending-bindings](https://laravel.com/docs/11.x/container#extending-bindings)

But now that I have extended the class to LivewireManager, as you had mentioned, any extra package that executes an extension to LivewireManager previously to Livewire-Discover and that overwrites the LivewireManager methods for any reason, will be lost, since my decorator uses the magic method "__call" to call those non-existent or overwritten methods of the last modified LivewireManager, but now it will only call those that do not exist in the original LivewireManager.

Do you or anyone else reading this know how I can fix this? Or is there any other magic method that allows me to call both non-existent and overwritten methods?

**Comment by pelmered** [comment]
Thank you for quick fix @joserick!

I can also confirm that it now works with version 1.3.1

I'm not sure why you need to inject the `LivewireManager` in the constructor and use that `__call` method.
